src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericUploadFile.js

```diff
@@ -82,7 +84,21 @@ GenericUploadFile.propTypes = {
   defaultParameterValue: PropTypes.any,
   resetParameterValue: PropTypes.func.isRequired,
   isDirty: PropTypes.bool,
+  error: PropTypes.object,
 };
 GenericUploadFile.defaultProps = {
   isDirty: false,
 };
+GenericUploadFile.useValidationRules = () => {
+  const { t } = useTranslation();
+  return {
+    validate: {
+      fileFormat: (value) => {
+        return (
+          FileManagementUtils.isFileFormatValid(value.file.type) ||
```

Comment:
Are we sure that `value` and `value.file` will never be `undefined`?
Maybe we should write `value?.file?.type` instead?

Reply:
The File [documentation](https://developer.mozilla.org/en-US/docs/Web/API/File/type) reads that `type` shouldn't be `undefined`, so if you're sure that RHF validation function won't be called with an undefined value, it should work :ok_hand: